compose/install.md

```diff
@@ -4,37 +4,52 @@ keywords: compose, orchestration, install, installation, docker, documentation
 title: Install Docker Compose
 ---
 
-You can run Compose on macOS, Windows and 64-bit Linux. To install it, you'll need to install Docker first.
+You can run Compose on macOS, Windows and 64-bit Linux.
 
-To install Compose, do the following:
+## Prerequisites
 
-1.  Install Docker Engine:
+You need to install Docker first, before you install and run Docker Compose.
 
-    * [Mac installation](/docker-for-mac/index.md){: target="_blank" class="_"}
+- On desktop systems like Docker for Mac and Windows, Docker Compose is
+included as part of those desktop installs. Skip down to [Install Compose on Linux systems](#install-compose-on-linux-systems).
 
-    * [Windows installation](/docker-for-windows/index.md){: target="_blank" class="_"}
+- On Linux systems, first install the
+[Docker server](/engine/installation/index.md#server){: target="_blank" class="_"}
```

Comment:
@shin- I think we are gently trying to deprecate it so I changed it to just "Docker"